packages/reporter/cypress/fixtures/retries_data.json

```diff
@@ -0,0 +1,274 @@
+{
```

Comment:
remove all of these fixtures, and just use isolated runner.

if we DID want to keep them, they'd only be useful for component tests, but they are not worth the upkeep or hassle